### ISSUE TITLE
Setup storybook deployment for frontend components

### ISSUE THREAD

**Issue by pavish** [centerofci/mathesar#308]
**Problem**
We have setup storybook for frontend component. We haven't deployed it anywhere yet.

**Proposed solution**
A GH action to build and deploy the storybook.

**Comment by pavish** [comment]
Blocked by PR #307

**Comment by pavish** [comment]
I'm going to close this issue, since it's not a priority and is something we'd only need after splitting our component library away from the app, probably as a separate repo, or even a mono-repo with a separate package.
